packages/aragon-cli/src/cli.js

```diff
@@ -109,7 +109,11 @@ cmd.option('apm.ipfs.rpc', {
   description: 'An URI to the IPFS node used to publish files',
   default: 'http://localhost:5001#default',
 })
-cmd.group('apm.ipfs.rpc', 'APM providers:')
+cmd.option('apm.ipfs.gateway', {
+  description: 'An URI to the IPFS Gateway to read files from',
+  default: 'http://localhost:8080/ipfs',
```

Comment:
Wondering if we should default this to `ipfs.eth.aragon.network/ipfs` instead, since most users who aren't running this in an aragon app directory (e.g. just using the `dao` commands to configure a dao) won't have an environment applied.